AnnoyancesFilter/sections/popups.txt

```diff
@@ -121,6 +121,7 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 !######### Subscription popups add to subscriptions.txt #########
 !
 techacademy.jp##.pum-overlay[data-popmake*="640px"]
+techcrunch.com##.announcement-banner[style="position: fixed;"]
```

Comment:
Is that `[style="position: fixed;"]` needed?

Reply:
This is part of the content and therefore in my opinion can be on the page

Reply:
Got it. Already found fixed banner.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/162482239-b0630633-be48-42aa-8975-cd0170371e69.png)

</details>